generated/src/aws-cpp-sdk-s3-crt/source/S3CrtClient.cpp

```diff
@@ -381,6 +381,10 @@ void S3CrtClient::init(const S3Crt::ClientConfiguration& config,
 
   static const size_t DEFAULT_PART_SIZE = 8 * 1024 * 1024; // 8MB
   s3CrtConfig.part_size = config.partSize < DEFAULT_PART_SIZE ? DEFAULT_PART_SIZE : config.partSize;
+  static const uint64_t FIVE_MB = 5 * 1024 * 1024;
+  s3CrtConfig.multipart_upload_threshold = config.multipartUploadThreshold.has_value() ?
+    config.multipartUploadThreshold.value() :
+    std::max(s3CrtConfig.part_size, FIVE_MB);
```

Comment:
dont set default and let crt default? passing 0 should be fine

Reply:
I wanted to have a bit of fun with with the logic too. But yeah makes sense will update